test/unit/spec/common/runtime-spec.js

```diff
@@ -21,19 +26,23 @@ define(['common/runtime', 'base/js/namespace', 'narrativeConfig'], (Runtime, Jup
             expect(Runtime.make).toBeDefined();
             expect(Runtime.make).toEqual(jasmine.any(Function));
         });
+
+        afterAll(() => TestUtil.clearRuntime());
     });
 
     describe('The runtime instance', () => {
         beforeAll(() => {
-            window.kbaseRuntime = null;
+            TestUtil.clearRuntime();
         });
 
         beforeEach(function () {
             this.runtime = Runtime.make();
         });
 
-        afterEach(() => {
-            window.kbaseRuntime = null;
+        afterEach(function () {
+            this.runtime.destroy();
```

Comment:
I believe the `this` context has already been destroyed by the time you get to the `afterEach` function. I don't know if you want to be extra safe and assign a `runtime` variable so you can explicitly run `runtime.destroy()`?

Reply:
Yeah, I'll do that. Didn't realize `this` is gone by the `afterEach` function. That's a little confusing.